### ISSUE TITLE
How does the open database with ... work?

### ISSUE THREAD

**Issue by AddictArts** [PostgresApp/PostgresApp#661]
The docs mention double clicking a database, but nothing happens for me? This is the only hint at what this preference means. Thanks for your help.

**Comment by AddictArts** [comment]
So, if you delete or drop the database named "username" or $USER no db's show up. Everything works like normal, but in the main window you will not see the cylinder or multiples of them for each db defined. Once I created that db again, they appeared. Then I could double click and open in psql and Postico. Even Postico2.

**Comment by jakob** [comment]
Yes, there should probably be some kind of warning when Postgres.app can't connect to the server. But I'm not sure how to write this warning, since it's not a critical error -- it's completely normal in some configurations that the GUI can't connect to the server. It doesn't necessarily mean there's something wrong with the server.

For example, another way this issue can come up is when you change auth settings to require a password to connect. Then the Postgres.app GUI can't connect to the server anymore.
